package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "leaflet-environmental-layers",
-  "version": "2.4.7",
+  "version": "2.4.8",
```

Comment:
Version bump here and in corresponding package-lock.json!

Reply:
Yup, I missed the bump in 2.4.7 for the lock file at https://github.com/publiclab/leaflet-environmental-layers/commit/702ebe1877ad9d816bd9856798607b325b7ffe13 😅